subprojects/docs/src/docs/userguide/userguide_single.adoc

```diff
@@ -50,6 +50,8 @@ include::gradle_wrapper.adoc[leveloffset=+2]
 
 include::troubleshooting.adoc[leveloffset=+2]
 
+include::directory_layout.adoc[leveloffset=+2]
```

Comment:
:o: Here I'd say this probably belongs just under build_environment.

Reply:
Do users really need to know this information to make good use of Gradle? I get by without most of it. I used to check the modules cache once upon a time, but I'm not sure even that is particularly useful now.

I would much prefer to see this treated as more of an appendix: a reference for those who may find this information interesting or helpful, but not something many users need to worry about.